src/natcap/invest/forest_carbon_edge_effect.py

```diff
@@ -842,7 +842,7 @@ def _calculate_tropical_forest_edge_carbon_map(
         # shape of distances and indexes: (x, k)
         distances, indexes = kd_tree.query(
             coord_points, k=n_nearest_model_points,
-            distance_upper_bound=DISTANCE_UPPER_BOUND, n_jobs=-1)
+            distance_upper_bound=DISTANCE_UPPER_BOUND, workers=-1)
```

Comment:
I think there are some comments above this call that reference `n_jobs`, could you update those?

Reply:
Also, does this mean we should set a lower bound on Scipy?